CHANGELOG.md

```diff
@@ -34,12 +34,12 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where the `.sav` file was not deleted upon exiting JabRef. [#6109](https://github.com/JabRef/jabref/issues/6109)
 - We fixed a linked identifier icon inconsistency. [#6705](https://github.com/JabRef/jabref/issues/6705)
 - We fixed the wrong behavior that font size changes are not reflected in dialogs. [#6039](https://github.com/JabRef/jabref/issues/6039)
+- We fixed the failure to Copy citation key and link. [#5835](https://github.com/JabRef/jabref/issues/5835)
 - We fixed an issue where the sort order of the entry table was reset after a restart of JabRef. [#6898](https://github.com/JabRef/jabref/pull/6898)
 - We fixed an issue where no longer a warning was displayed when inserting references into LibreOffice with an invalid "ReferenceParagraphFormat". [#6907](https://github.com/JabRef/jabref/pull/60907).
 
 ### Removed
 
-## [5.1] – 2020-08-30
```

Comment:
Please do not delete CHANGELOG headers

Reply:
Sorry about that! No idea how that happened...

Reply:
I think, it happend during conflict resolution when merging the master branch: https://github.com/JabRef/jabref/pull/6871/commits/d022875ba0e9e02e16722b73897126c8fd79946b

Could you just re-add the line in another commit?

Reply:
quickfix